lithops/standalone/backends/ibm_vpc/ibm_vpc.py

```diff
@@ -314,15 +314,29 @@ def delete_instance(instance_info):
                 else:
                     raise e
 
+        LITHOPS_MASTER = 'lithops-master-'
+
+        vms_prefixes = ('lithops-worker',)
+        if self.config.get('delete_master'):
+            vms_prefixes = vms_prefixes + (LITHOPS_MASTER, )
+
         deleted_instances = set()
         while True:
             instances_to_delete = set()
             instances_info = self.ibm_vpc_client.list_instances().get_result()
             for ins in instances_info['instances']:
-                if ins['name'].startswith('lithops-worker'):
+                if ins['name'].startswith(vms_prefixes):
                     ins_to_dlete = (ins['name'], ins['id'])
                     if ins_to_dlete not in deleted_instances:
                         instances_to_delete.add(ins_to_dlete)
+                    if ins['name'].startswith(LITHOPS_MASTER):
+                        # clean all been triggered, delete also master floating IP
+                        interface_id = ins['network_interfaces'][0]['id']
+                        fips = self.ibm_vpc_client.list_instance_network_interface_floating_ips(
+                            ins['id'], interface_id).get_result()['floating_ips']
+                        if fips:
+                            fip = fips[0]['id']
+                            self.ibm_vpc_client.delete_floating_ip(fip)
```

Comment:
Not sure if you already took this into account, but note that floating IPs are not serverless: When a user requests a new floating IP, he pays 1$ for the whole month, even if he uses the IP for only one minute. So, if someone during a month runs 100 times the command `lithops clean -b ibm_vpc --all`, running Lithops after this each time, he will pay 100$ only for IP addresses.

Reply:
> Yep, this makes more sense

done